spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -28,6 +28,27 @@
     end
   end
 
+  describe 'attributes.hesa_itt_data' do
+    let(:application_form) do
+      create(:completed_application_form, :with_completed_references)
+    end
+
+    let(:application_choice) do
+      create(:application_choice, status: 'awaiting_provider_decision', application_form: application_form)
+    end
+
+    it 'is hidden by default' do
+      response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
+      expect(response.dig(:attributes, :hesa_itt_data)).to be_nil
+    end
+
+    it 'becomes available once application status is \'enrolled\'' do
+      application_choice.update_attributes(status: 'enrolled')
+      response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
+      expect(response.dig(:attributes, :hesa_itt_data)).not_to be_nil
```

Comment:
Perhaps we can add a check to see if the response conforms to the schema here, I think it wouldn't be tested otherwise.

```
expect(response.to_json).to be_valid_against_openapi_schema('Application')
```